src/components/Form.tsx

```diff
@@ -10,7 +10,7 @@ import {
 } from '@mui/material';
 import { Container } from '@mui/system';
 import React from 'react';
-import solidDarkBackground from '../../static/hero-solid-dark.jpg';
+import solidLightBackground from '../../public/hero-solid-light.jpg';
```

Comment:
This file miss in the repo (and maybe the right place is in the `static` folder instead of `public` )